files/fetch_store_patch.rb

```diff
@@ -0,0 +1,15 @@
+module ActionDispatch
+  class Request
+    class Session
+      alias_method :fetch, :[]
+      alias_method :store, :[]=
+    end
+  end
+  
+  class Cookies
+    class CookieJar
+      alias_method :fetch, :[]
+      alias_method :store, :[]=
+    end
+  end
+end
```

Comment:
There seems to be a little line ending warning here.

Reply:
Good catch!